cmake/dependency.dav1d.cmake

```diff
@@ -66,7 +66,10 @@ if (MSVC)
     set(DAVID_DEBUG_LIBRARY libdav1d_deb)
 
 elseif(UNIX)
-     string(TOLOWER ${CMAKE_BUILD_TYPE} CMAKE_BUILD_TYPE_FOR_STUPID_MESON)
+     set(CMAKE_BUILD_TYPE_FOR_STUPID_MESON "plain")
```

Comment:
I would use "debug" by default. In my experience it is usually used as default.
Also it's just enough to check `if(CMAKE_BUILD_TYPE)`